docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -701,14 +704,14 @@ Then, we add an 'Authentication rules':
 * *Description:* Rule for corporate machines
 * Don't set any condition (as it's a catch-all rule)
 * Set the following *actions:*
-  - Set role corporate_machine
-  - Set unregistration date January 1st, 2020
+  * Role - corporate_machine
+  * Unregistration date - January 1st, 2020
```

Comment:
I will make more sense to use Access duration here in place of using Unregistration date with a wrong value.